### PR TITLE
chore: improve the inspect query for front-end

### PR DESCRIPTION
**Pull request by gsk967**
## Description

added two new fields two the existed inspect query response 
1. base_denom -> base denom of coin (Ex: uumee for UMEE , ibc/XXX for remaining tokens )
2. base_amount -> base amount of coin without convert to SYMBOL Denom value

---

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 - [2494](https://github.com/umee-network/umee/pull/2494) Use go 1.22
 - [2495](https://github.com/umee-network/umee/pull/2495) (util) `KeyWithUint64` and `KeyWithUint32` helper functions for db keys.
 - [2504](https://github.com/umee-network/umee/pull/2504) bump `wasmd` to v0.45.0.
+- [2562](https://github.com/umee-network/umee/pull/2562) (x/leverage): Improved the existing inspect account response by adding `base_denom` and `base_amount`.
```
Comment (robert-zaremba) [inline CHANGELOG.md:63]:
```suggestion
- [2562](https://github.com/umee-network/umee/pull/2562) (x/leverage proto): Add `base_denom` and `base_amount` to `QueryInspectResponse`.
```

File: x/leverage/keeper/inspector.go [hunk 2]
```diff
@@ -111,7 +114,21 @@ func (q Querier) Inspect(
...
+				}
+			}
+
+			for _, c := range borrowers[j].Position.Borrowed {
+				if c.Denom == req.Symbol {
+					b = c.BaseAmount
+					break
+				}
+			}
+			return a.GTE(b)
```
Comment (robert-zaremba) [inline x/leverage/keeper/inspector.go:131]: what i `a` or `b` is undefined? Let's add a unit test if possible.

**Review by robert-zaremba — COMMENTED** [review]
let's add one test to cover the edge case as noted in the comment.

**Review by robert-zaremba — APPROVED** [review]
